dashboard/templates/base.html

```diff
@@ -66,6 +66,22 @@
 <script src="{% static 'jquery/dist/jquery.min.js' %}"></script>
 <script src="{% static 'angular/angular.js' %}"></script>
 <script src="{% static 'underscore/underscore-min.js' %}"></script>
+<script>
+    $(document).ready(function () {
+        url = window.location.href;
+        var help_link='https://sites.google.com/umich.edu/student-dashboard-help/home';
```

Comment:
seems to me that we can define the base url in a variable. otherwise, it all looks good!

Reply:
Yeah right, help_link is kind of the base, in all of the methods you can just go

help_link=help_link+"/grade-distribution" or whatever the link is for

I'd also make this into an else if just incase it matches multiple cases